examples/CloudFunction/README.md

```diff
@@ -0,0 +1,121 @@
+- Admin SDK API: `admin.googleapis.com`
+- PubSub API: `pubsub.googleapis.com`
+- Cloud Functions API: `cloudfunctions.googleapis.com`
+- Cloud Scheduler API: `cloudscheduler.googleapis.com`
+- App Engine API: `appengine.googleapis.com`
+
+
+## Prerequisites 
+
+The following GSuite Admin APIs are currently supported:
+
+- `reports_v1` - [Reports API](https://developers.google.com/admin-sdk/reports/v1/get-start/getting-started)
+    - `admin` - [Admin activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-admin.html)
+    - `drive` - [Google Drive activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-drive.html)
+    - `login` - [Login activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-login.html)
+    - `mobile` - [Mobile activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-mobile.html)
+    - `token` - [Authorization Token activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-tokens.html)
+
+
+The following destinations are currently supported:
+- [Stackdriver Logging](https://cloud.google.com/logging/docs/)
+
+### Service account
+We need two Terraform service accounts for this module:
+* **Terraform service account** (that will create the Scheduler & Cloudfunction)
+* **CloudFunction service account** (that will be used by the CloudFunction to pull logs from GSuite and write them to Stackdriver Logging)
+
+The **Terraform service account** used to run this module must have the following IAM Roles:
+- `PubSub Admin` on the project to create the pubsub
+- `Storage Admin` on the project to create the storage accounts
+- `Service Account User` on the project (to associate the Cloudfunction service account with the CloudFunction)
+- `Cloud Scheduler Admin` on the project to create the cloud scheduler
+- `CloudFunction Admin` on the project to create the cloud function.
+
+The **CloudFunction service account** passed to this module must have :
+- A GSuite Admin account
+- A service account with:
+  * [GSuite domain-wide delegation](https://developers.google.com/admin-sdk/reports/v1/guides/delegation) enabled.
+  * The IAM roles
+    - `roles/iam.serviceAccountTokenCreator` set on the organization.
+    - `roles/logging.viewer` set on the project to view logs
+    - `roles/logging.configWriter` set on the project to write logs to stackdriver
+
+
+  * The following scopes in the [API client access page](https://admin.google.com/AdminHome?chromeless=1#OGX:ManageOauthClients)
+    - https://www.googleapis.com/auth/admin.reports.audit.readonly (to read from the Reports API)
+    - https://www.googleapis.com/auth/iam (to generate a super-admin token)
+
+## Usage
+The usage of the module within your own main.tf file is as follows:
+
+```hcl
+    module "aggregated-export-to-pubsub" {
+      source                          = ./path-to-your-source
```

Comment:
`./path-to-your-source` --> `terraform-google-modules/gsuite-export/google`

Add:
`version = "~> 1.0.0"`

Reply:
```suggestion
      source                          = "terraform-google-modules/gsuite-export/google"
```